docs/writing-stories/index.mdx

```diff
@@ -100,6 +100,103 @@ components/
 
 {/* prettier-ignore-end */}
 
+<If renderer="svelte">
+
+<details>
+<summary>Alternative story definitions</summary>
+ 
+For most cases, we recommend Svelte CSF users define their stories as above, which uses the default rendering of the component. However, there are alternatives to this approach.
+
+Any `children` elements passed to the Story component will be passed along as `args.children` to the component:
```

Comment:
note: only if there is a `component` set in `defineMeta`. if `defineMeta` is `component`-less, `Story`'s `children` will be rendered as-is, ie. the `asChild` prop will implicitly be true in that case.

It might sound a little confusing, and be confusing to document. but it makes sense that you can't really forward `children` to a component if no component is specified.